linkerd/assets/dashboards/-_overview.json

```diff
@@ -0,0 +1,771 @@
+{
+    "author_info": {
+        "author_name": "Datadog"
+    },
+    "board_bgtype": "board_graph",
+    "board_title": "Linkerd - Overview",
+    "created": "2020-01-06T22:20:03.614862+00:00",
+    "created_by": {
+        "access_role": "adm",
+        "disabled": false,
+        "email": "support@datadoghq.com",
+        "handle": "support@datadoghq.com",
+        "is_admin": true,
+        "name": "Datadog",
+        "role": null,
+        "verified": true
+    },
+    "description": "## Linkerd\n\nThis dashboard provides a high-level overview of your Linkerd cluster so you can monitor its performance and route, control, and connection metrics.\n\nFor further reading on monitoring Envoy, see:\n\n- [Official Linkerd integration docs](https://docs.datadoghq.com/integrations/linkerd/)\n\nClone this template to make changes and add your own graphs and widgets.",
+    "disableCog": false,
+    "disableEditing": false,
+    "icon": "/static/images/saas_logos/small/istio.png",
```

Comment:
```suggestion
    "icon": "/static/images/saas_logos/bot/linkerd@2x.png",
```

Reply:
The @2x suffix is automatically added by the backend actually.